src/app/(proper_react)/redesign/(public)/SignUpForm.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../tokens";
+
+.form {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $spacing-xl;
+
+  &.isHero {
```

Comment:
I don't see this class on the form element?

(But it might still be nice to add it, instead of adding it directly on the `input` and `button`, and then move the hero-specific styles for those elements into this block - then all hero-specific styles are together. Although I also see the argument for keeping input- and button-related styles together, so up to you.)